tests/test_configurations/test_base.py

```diff
@@ -427,6 +427,21 @@ def test_public_id_invalid_version():
         PublicId("author", "name", object())
 
 
+def test_public_id_from_string():
+    """Test parsing the public id from string."""
+    public_id = PublicId.from_str("author/package:0.1.0")
+    assert public_id.author == "author"
+    assert public_id.name == "package"
+    assert public_id.version == "0.1.0"
+
+
+def test_public_id_from_string_without_version_string():
+    public_id = PublicId.from_str("author/package")
```

Comment:
what about allowing the alias `autor/package:latest` as well?

Reply:
we do